### PR TITLE
Improvements of Modules lesson according to feedback

### PR DESCRIPTION
**Pull request by kochaika**
Feedback: [EDC-996](https://youtrack.jetbrains.com/issue/EDC-996), [EDC-997](https://youtrack.jetbrains.com/issue/EDC-997)

Changes description:
- Fixed order of files in task-info.yaml to be main task file opened when task in choosen.
- Splited first task of lesson `Import module` into two tasks to lighten the threshold of entry and make the first task of this module as easy as possible.
- Some minor сlarifications to the task descriptions

### CODE REVIEW

**Review by Geravant — APPROVED** [review]

File: Modules and packages/Import module 2/task.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Import module 2
+
+You can import not only functions: you can import classes or even other modules. It is customary but not required to place all
```
Comment (stephen-hero) [inline Modules and packages/Import module 2/task.md]: You can import not only functions but also classes and even other modules. It is customary, though not required, to place all

File: Modules and packages/Import module 2/task.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Import module 2
+
+You can import not only functions: you can import classes or even other modules. It is customary but not required to place all 
+import statements at the beginning of a module.
+
+You can find out more about modules in Python by reading [this section](https://docs.python.org/3/tutorial/modules.html) of The Python Tutorial.
```
Comment (stephen-hero) [inline Modules and packages/Import module 2/task.md]: You can learn more about modules in Python by reading [this section](https://docs.python.org/3/tutorial/modules.html) of The Python Tutorial.

File: Modules and packages/Import module 2/task.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+### Task
+In the code editor, import the module `calculator` and create an instance of the class `Calculator` (`calc`).
```
Comment (stephen-hero) [inline Modules and packages/Import module 2/task.md]: In the code editor, import the `calculator` module and create an instance of the `Calculator` class (`calc`).

File: Modules and packages/Import module/task-info.yaml [hunk 4]
```diff
@@ -1,23 +1,15 @@
...
-- name: tests/test_task.py
-  visible: false
-feedback_link: https://docs.google.com/forms/d/e/1FAIpQLSfRlDlldKfuq-cHMNFfHMER61P1PRIan7KG6yp1GvaweDI7GA/viewform?usp=pp_url&entry.2103429047=Modules+and+Packages+/+Import+module
\ No newline at end of file
+  - name: imports.py
+    visible: true
+    placeholders:
+      - offset: 26
+        length: 19
+        placeholder_text: '# call hello_world function from the my_funcs module'
```
Comment (stephen-hero) [inline Modules and packages/Import module/task-info.yaml]: placeholder_text: '# call the hello_world function from the my_funcs module'

File: Modules and packages/Import module/task.md [hunk 5]
```diff
@@ -16,19 +16,13 @@ directly, but using the module name, you can now access the functions, for examp
...
-import statements at the beginning of a module.
-
-You can find out more about modules in Python by reading [this section](https://docs.python.org/3/tutorial/modules.html) of The Python Tutorial. 
   
 For more structured and detailed information, you can also refer to [this Hyperskill knowledge base page](https://hyperskill.org/learn/step/6019#module-loading).
 
 ### Task
-In the code editor, import the module `calculator` and create an instance of the class `Calculator` (`calc`). 
-Use the `add` method defined in `Calculator` in a loop to add up numbers from 0 to 99.
+In the code editor, you have already imported module `my_funcs`.
```
Comment (stephen-hero) [inline Modules and packages/Import module/task.md:23]: In the code editor, you have already imported the module `my_funcs`.

File: Modules and packages/Import module/task.md [hunk 6]
```diff
@@ -16,19 +16,13 @@ directly, but using the module name, you can now access the functions, for examp
...
-
-You can find out more about modules in Python by reading [this section](https://docs.python.org/3/tutorial/modules.html) of The Python Tutorial. 
   
 For more structured and detailed information, you can also refer to [this Hyperskill knowledge base page](https://hyperskill.org/learn/step/6019#module-loading).
 
 ### Task
-In the code editor, import the module `calculator` and create an instance of the class `Calculator` (`calc`). 
-Use the `add` method defined in `Calculator` in a loop to add up numbers from 0 to 99.
+In the code editor, you have already imported module `my_funcs`. 
+Call the function `hello_world` from this module with argument `"John"`
```
Comment (stephen-hero) [inline Modules and packages/Import module/task.md:24]: Call the function `hello_world` from this module with the argument `"John"`.

File: Modules and packages/Import module/tests/test_task.py [hunk 7]
```diff
@@ -11,25 +10,11 @@
...
-                                msg='`calc` should be an instance of Calculator.')
-            except NameError:
-                self.assertTrue(False, msg='Do not change variable names.')
-
         def test_out(self):
             expected, actual = 'Hello, World! My name is John', output[0]
-            self.assertEqual(expected, actual, msg='Please do not change the starter code.')
-            expected, actual = str(4950), output[1]
-            self.assertEqual(expected, actual, msg='Calculation result looks wrong.')
+            self.assertEqual(expected, actual, msg='Call hello_world with "John" argument')
```
Comment (stephen-hero) [inline Modules and packages/Import module/tests/test_task.py]: self.assertEqual(expected, actual, msg='Call hello_world with the "John" argument')

File: Modules and packages/Import module/tests/test_task.py [hunk 8]
```diff
@@ -11,25 +10,11 @@
...
+except AttributeError:
     class TestFailCase(unittest.TestCase):
         def test_fail(self):
-            self.assertTrue(False, msg='You need to import the calculator module.')
-
-except ModuleNotFoundError:
-    class TestFailCase1(unittest.TestCase):
-        def test_fail(self):
-            self.assertTrue(False, msg="Don't use file extensions in imports.")
+            self.assertTrue(False, msg='You need to use hello_world function from my_funcs module.')
```
Comment (stephen-hero) [inline Modules and packages/Import module/tests/test_task.py]: self.assertTrue(False, msg='You need to use the hello_world function from the my_funcs module.')

File: Modules and packages/Packages/task.md [hunk 9]
```diff
@@ -43,6 +43,8 @@ You can learn more about packages by reading <a href="https://docs.python.org/3/
 For more structured and detailed information, you can also refer to [this Hyperskill knowledge base page](https://hyperskill.org/learn/step/6384).
 
 ### Task
+Look at the file structure in the `classes` and `functions` directories and in the subdirectories.
```
Comment (stephen-hero) [inline Modules and packages/Packages/task.md:46]: Look at the file structure in the `classes` and `functions` directories and their subdirectories.

**Review by stephen-hero — CHANGES_REQUESTED** [review]

File: Modules and packages/Import_module_part_two/task-info.yaml [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+files:
+  - name: imports.py
+    visible: true
+    placeholders:
+      - offset: 0
+        length: 17
+        placeholder_text: '# Import the `calculator` module here'
+      - offset: 26
+        length: 23
+        placeholder_text: '''Create a new instance of Calculator class defined in calculator'''
```
Comment (stephen-hero) [inline Modules and packages/Import_module_part_two/task-info.yaml:12]: '''Create a new instance of the Calculator class defined in calculator'''

**Review by stephen-hero — CHANGES_REQUESTED** [review]

**Review by DaniilBogdanovJB — APPROVED** [review]

**Review by stephen-hero — APPROVED** [review]

**Pull request merged**